### PR TITLE
Add CSS-in-JS micro badge to React apprentice

### PR DESCRIPTION
**Pull request by JHStoops**
## Changes

- Add CSS-in-JS as a micro badge in React's Apprentice badge.
	- This badge focuses on FamilySearch's CSS-in-JS best practices found [here](https://www.familysearch.org/frontier/docs/develop/css-best-practices)
	- I added an option for completion to change an existing app or shared component to conform with our best practices as a call to action.

### CODE REVIEW

**Review by marybeshaw — COMMENTED** [review]
This is really nice!

**Comment by JHStoops** [comment]
Let me know if you think there’s anything missing that would make it a better micro badge. With the other two micro badges being ~3hrs each, I just wanted to keep this at ~1hr. Also, I tried finding more videos, but the ones I found that used the css attribute instead of styled components had poor audio quality

File: badges-active/react/_micro_css-in-js.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- You should have some experience working with CSS-in-JS, scoping styles, and applying FamilySearch's CSS-in-JS best practices.
+
+-----
+
+## Intro
+
+> If you are using a JavaScript framework to build a web app with components, CSS-in-JS is probably a good fit. Especially if you are part of a team where everybody understands basic JavaScript.
+> [Max Stoiber](https://mxstbr.com/thoughts/css-in-js/)
+
+{brief description of topic and why it's important}
```
Comment (spacerockzero) [inline badges-active/react/_micro_css-in-js.md:19]: placeholder text formatted like `{placeholdertext}` can be deleted

**Review by spacerockzero — APPROVED** [review]
if you remove that placeholder text, it looks ready to merge

File: badges-active/react/_micro_css-in-js.md (hunk 1 above)
Comment (JHStoops) [reply badges-active/react/_micro_css-in-js.md:19]: Oops! I should probably put a quick description in its place. Good catch! Thanks for approving this PR. I'll add that in and merge it :)

Comment (JHStoops) [reply badges-active/react/_micro_css-in-js.md]: I just used the [reasoning they used for using Emotion](https://www.familysearch.org/frontier/docs/architectural-decisions/006-emotion) as the Introduction to this micro badge.

**Pull request merged**